### ISSUE TITLE
How could I predict the test set?

### ISSUE THREAD

**Issue by extremin** [kingfengji/gcForest#1]
I'm not talking about the validation set with golden labels, but the test set.
When I have trained the model, I want to predict the test set. But I could not find the predict function or any configuration.

**Comment by kingfengji** [comment]
sure you can.( test accuracy is based on the predictions of test set, run any demo scripts to see it)
 
Check out train.cascade.cascade.data_save_dir in model config files, where this is the dir for predictions of both training and testing data. If your test data does not have labels, make some fake labels and ignore the test accuracy(since you'll never know the real test acc.)

**Comment by extremin** [comment]
Thx for your reply.
another questions: What does tools/train_xgb.py mean?

**Comment by kingfengji** [comment]
Meaning you could train a xgboost on top of gcForest. You could also train other models by similar procedure. Also the base classifier can be changed accordingly.
